CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 * Fixed #505 – Work around an issue in Chrome 57 where dynamically inserted HTML Imports did not subsequent script execution. See https://bugs.chromium.org/p/chromium/issues/detail?id=701601
```

Comment:
dynamically inserted HTML Imports "did not subsequent script execution." <-- grammar-wut?